docs/next/modules/en/pages/operator/airgapped.adoc

```diff
@@ -18,7 +18,179 @@ To install Cluster API providers in an air-gapped environment the following will
  ** Provide fetch configuration and image values for `core` and `caprke2` providers in xref:../operator/chart.adoc#_cluster_api_operator_values[values.yaml].
  ** Provider image value for the Cluster API Operator Helm chart dependency in https://github.com/kubernetes-sigs/cluster-api-operator/blob/main/hack/charts/cluster-api-operator/values.yaml#L26[values.yaml]. Image values specified with the cluster-api-operator key will be passed along to the Cluster API Operator.
 
-== Example Usage
+== CAPI Provider installation with OCI artifact
+
+As an administrator working in an air-gapped environment, you need to fetch the Azure Cluster API Provider (CAPZ) components from within your cluster since external internet access is restricted. This section demonstrates how to deploy CAPZ using OCI artifacts stored in your private registry.
+
+Set your private registry URL and replace with your actual registry:
+
+[source,bash]
+----
+export REGISTRY=<YOUR_PRIVATE_REGISTRY>
+export RELEASE_TAG=<CAPI_PROVIDER_VERSION>
+----
+
+Create and apply Azure CAPIProvider resource that instructs Rancher Turtles to fetch the Azure provider from your private OCI registry:
+
+.capz-provider-oci.yaml
+[source,yaml]
+----
+apiVersion: turtles-capi.cattle.io/v1alpha1
+kind: CAPIProvider
+metadata:
+  name: azure
+  namespace: capz-system
+spec:
+  type: infrastructure
+  name: azure
+  version: ${RELEASE_TAG}
+  fetchConfig:
+    oci: ${REGISTRY}:${RELEASE_TAG}
+----
+
+== How to mirror CAPI Provider OCI artifact
```

Comment:
I think this section can be replaced with https://cluster-api-operator.sigs.k8s.io/03_topics/03_plugin/03_publish_subcommand#publishing-multiple-providers-and-versions-in-an-oci-image

Reply:
I have added it